Lib/gettext.py

```diff
@@ -46,11 +46,9 @@
 #   find this format documented anywhere.
 
 
-import copy
 import locale
 import os
 import re
-import struct
 import sys
 from errno import ENOENT
```

Comment:
Would it be worth it to defer errno import as well? It's only used in one case, to raise an error when something goes wrong.

Reply:
It is imported by `os` (where it is only used in one case too).